src/main/java/crud/dao/impl/ManufacturerDaoJdBcImpl.java

```diff
@@ -0,0 +1,145 @@
+                    + throwables);
+        }
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String query = "SELECT id, name, country FROM manufacturers WHERE id = "
+                + id + " AND deleted = FALSE";
+
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            Statement statement = connection.createStatement();
+            ResultSet resultSet = statement.executeQuery(query);
+            Manufacturer manufacturer = new Manufacturer("", "");
+            while (resultSet.next()) {
+                Long newId = resultSet.getObject("id", Long.class);
+                manufacturer.setId(newId);
+                String name = resultSet.getString("name");
+                manufacturer.setName(name);
+                String country = resultSet.getString("country");
+                manufacturer.setCountry(country);
+            }
+            statement.close();
+
+            return Optional.ofNullable(manufacturer);
+        } catch (SQLException throwables) {
+            throw new RuntimeException("Couldn't establish connection to MySQL server. "
+                    + throwables);
+        }
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        String query = "SELECT * FROM manufacturers WHERE deleted = FALSE";
+
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            List<Manufacturer> manufacturers = new ArrayList<>();
+            Statement statement = connection.createStatement();
+            ResultSet resultSet = statement.executeQuery(query);
+            while (resultSet.next()) {
+                Long id = resultSet.getObject("id", Long.class);
+                String name = resultSet.getString("name");
+                String country = resultSet.getString("country");
+                Manufacturer manufacturer = new Manufacturer(name, country);
+                manufacturer.setId(id);
+                manufacturers.add(manufacturer);
+            }
+            statement.close();
+
+            return manufacturers;
+        } catch (SQLException throwables) {
+            throw new RuntimeException("Couldn't establish connection to MySQL server. "
+                    + throwables);
+        }
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String query = "UPDATE manufacturers SET name = ?, country = ?"
+                + " WHERE id = ? AND deleted = FALSE";
+        String getNewQuery = "SELECT * FROM manufacturers WHERE id = " + manufacturer.getId();
+
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement preparedStatement
+                    = setUpdate(connection.prepareStatement(query), manufacturer);
+            preparedStatement.setLong(3, manufacturer.getId());
+            preparedStatement.executeUpdate();
+            preparedStatement.close();
```

Comment:
Honestly, that`s enough for this method) you can return manufacturer, that you already have